views/project/billing.erb

```diff
@@ -142,6 +142,26 @@
                     }
                   ) %>
                 </div>
+                <div class="col-span-full">
+                  <%== render(
+                    "components/form/text",
+                    locals: {
+                      name: "tax_id",
+                      label: "Tax Id (Optional)",
+                      value: @billing_info_data[:tax_id]
+                    }
+                  ) %>
+                </div>
+                <div class="col-span-full">
+                  <%== render(
+                    "components/form/text",
+                    locals: {
+                      name: "company_name",
+                      label: "Company Name (Optional)",
```

Comment:
Let's don't put `(Optional)`. The fields are already empty as it's an update form. Instead, I prefer using `*` for required fields, rather than the lengthy `Optional` text. But it's not required for this PR.

Reply:
Yep, that makes sense. Let me add it with a subsequent PR as I don't want to add * to each one but maybe create a new tag/class. So, I'm resolving the comment but added a task to my list.